### PR TITLE
[dev] BUG AB#36859 [Events] Recurring Events Not Working Correctly

### PR DESCRIPTION
**Pull request by IlyaRadinsky**
Fixes handling click on prev/next month buttons

### CODE REVIEW

**Comment by IlyaRadinsky** [comment]
@morethanfire Please take a look.

**Review by groberts314 — APPROVED** [review]
Looks good to me.  Will wait for @morethanfire 's approval as well.

**Review by morethanfire — APPROVED** [review]
Looks good.

**Pull request merged**
